src/System.Private.ServiceModel/tools/test/Bridge/Bridge/IdleTimeoutManager.cs

```diff
@@ -35,12 +35,14 @@ internal static void Register(HttpConfiguration config)
         protected async override Task<HttpResponseMessage> SendAsync(
             HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            Debug.WriteLine("DEBUG: Processing : " + request.RequestUri);            
+            Trace.WriteLine(String.Format("Bridge received {0} : {1} at {2:T}", request.Method, request.RequestUri, DateTime.Now), 
+                            this.GetType().Name);            
             using (_timeoutManager.Start())
             {
                 // Call the inner handler.
                 var response = await base.SendAsync(request, cancellationToken);
-                Debug.WriteLine("DEBUG: Completed  : " + request.RequestUri);
+                Trace.WriteLine(String.Format("Bridge completed {0} : {1} at {2:T}", request.Method, request.RequestUri, DateTime.Now), 
+                                this.GetType().Name);
```

Comment:
NIT: Consider, when you have times involved in a log message, it would be nice to see them at the start of the line. For example, instead of seeing 

```
IdleTimeoutHandler: Bridge received PUT : http://localhost:44283/resource/ at 3:05:12 PM
IdleTimeoutHandler: Bridge completed PUT : http://localhost:44283/resource/ at 3:05:12 PM
```

I'd rather see: 

```
IdleTimeoutHandler: 3:05:12 PM - Bridge received PUT : http://localhost:44283/resource/ 
IdleTimeoutHandler: 3:05:12 PM - Bridge completed PUT : http://localhost:44283/resource/ 
```